data/json/recipes/recipe_food.json

```diff
@@ -8317,5 +8321,53 @@
       [ [ "sweet_fruit_like", 1, "LIST" ] ],
       [ [ "sugar", 5 ] ]
     ]
-  }
+  },
+  {
+    "type": "recipe",
+    "result": "sourdough_young",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREAD",
+    "skill_used": "cooking",
+    "difficulty": 6,
```

Comment:
Difficulty 6 for crafting a sourdough? We have much more complex recipes (like soups, sausages, spaghetti and others) at difficulty 4 and lower.

Reply:
Making sourdough bread is pretty easy, reflected in that recipe. Making the actual starter is pretty tricky though, requiring a fair bit of attention to timing, what you feed it, and the health of the culture for a while... As well as a little luck. Rather than require the player to actually do all that, I felt it best reflected in the difficulty of the recipe.

Reply:
After sleeping on it, I dropped it to 5. It is harder to get a good sourdough starter than it is to make a good stew, but not quite as hard as chemistry. Seems about right.